src/spaces/continuous.jl

```diff
@@ -101,7 +101,7 @@ end
 
 "given position in continuous space, return cell coordinates in grid space."
 pos2cell(a::AbstractAgent, model::ABM) = pos2cell(a.pos, model)
-pos2cell(pos::ValidPos, model::ABM) = Tuple(floor.(Int, pos./abmspace(model).spacing) .+ 1)
+pos2cell(pos::ValidPos, model::ABM) = Tuple(max.(1, ceil.(Int, pos./abmspace(model).spacing)))
```

Comment:
Is the `max(1, ...)` necessary since we do ceiling? is it even possible for the computation to return an integer 0? That would mean that the `pos ./ ... ` would return a negative number. Which isn't possible.

Reply:
See https://github.com/JuliaDynamics/Agents.jl/issues/853#issuecomment-1688351895

Reply:
actually, `pos` can be 0 in continuous space. so I guess we do need the `max`. Oh man, this is so expensive though. We call the `max` function for any position, even though the probability of having exactrly 0 is so small. Maybe it leads to better performance to first do a check of `newpos = ceil...` and then `any(iszero, pos)` and if `true` then do the `max.(1, newpos)`?

Reply:
I know, I'm also upset about this :D 
But it seems not to have actually any impact at all, consider that we have more costly operations inside the functions which call it, and also using `ceil` actually spares the `+1` we needed when we used it before